### DISCUSSION TITLE
Music control in navigation mode

### DISCUSSION THREAD

**Discussion by erikd256** [organicmaps/organicmaps#9866] [Ideas]
Hi,

I wanted starting off with building a maybe usefull feature, adding music control switches in the navigation pane. Kind of same as google maps and other apps have.

**Comment by erikd256** [comment]
Started building a layout into the navigation view. view the progress here: [https://github.com/erikd256/organicmaps](https://github.com/erikd256/organicmaps)

**Reply by hb0nd** [reply]
I'd recommend uploading some concept designs so it can be discussed before spending lots of time developing something. Nonetheless, thank you for working on it!

**Reply by erikd256** [reply]
Definitly true, I will start on some ideas

**Reply by erikd256** [reply]
Just pushed a first minimalist layout option to git

https://github.com/erikd256/organicmaps/commit/87f51cf4bdeb1a598cbb10f4f77a8cde8b2a3b08

**Reply by hb0nd** [reply]
It looks nice! let's move discussion to https://github.com/organicmaps/organicmaps/issues/1336 :)
